src/main/scala/net/ceedubs/sbtctags/SbtCtags.scala

```diff
@@ -31,26 +32,25 @@ object CtagsKeys {
 object SbtCtags extends Plugin {
 
   override val projectSettings = Seq(
-      CtagsKeys.dependencySrcUnzipDir <<= Keys.target (_ / "sbt-ctags-dep-srcs"),
+    CtagsKeys.dependencySrcUnzipDir <<= Keys.target(_ / "sbt-ctags-dep-srcs"),
 
-      CtagsKeys.ctagsParams in ThisBuild := defaultCtagsParams,
+    CtagsKeys.ctagsParams in ThisBuild := defaultCtagsParams,
 
-      CtagsKeys.ctagsSrcFileFilter <<= CtagsKeys.ctagsParams(_.languages.foldLeft(NameFilter.fnToNameFilter(_ => false))((filter, lang) => filter | GlobFilter(s"*.$lang"))),
+    CtagsKeys.ctagsSrcFileFilter <<= CtagsKeys.ctagsParams(_.languages.foldLeft(NameFilter.fnToNameFilter(_ => false))((filter, lang) => filter | GlobFilter(s"*.$lang"))),
 
-      CtagsKeys.ctagsSrcDirs <<= (Keys.scalaSource in Compile, Keys.scalaSource in Test, Keys.javaSource in Compile, Keys.javaSource in Test, CtagsKeys.dependencySrcUnzipDir){ (srcDir, testDir, javaSrcDir, javaTestDir, depSrcDir) =>
-        Seq(srcDir, testDir, javaSrcDir, javaTestDir, depSrcDir)
-      },
+    CtagsKeys.ctagsSrcDirs <<= (Keys.scalaSource in Compile, Keys.scalaSource in Test, Keys.javaSource in Compile, Keys.javaSource in Test, CtagsKeys.dependencySrcUnzipDir) { (srcDir, testDir, javaSrcDir, javaTestDir, depSrcDir) =>
+      Seq(srcDir, testDir, javaSrcDir, javaTestDir, depSrcDir)
+    },
 
       CtagsKeys.ctagsGeneration in ThisBuild := defaultCtagsGeneration(Keys.baseDirectory.value) _,
 
-      CtagsKeys.genCtags <<= (Keys.state, CtagsKeys.dependencySrcUnzipDir, CtagsKeys.ctagsParams, CtagsKeys.ctagsSrcFileFilter, CtagsKeys.ctagsGeneration, CtagsKeys.ctagsSrcDirs, Keys.streams) map genCtags
-  )
+    CtagsKeys.genCtags <<= (Keys.thisProjectRef, Keys.state, CtagsKeys.dependencySrcUnzipDir, CtagsKeys.ctagsParams, CtagsKeys.ctagsSrcFileFilter, CtagsKeys.ctagsGeneration, CtagsKeys.ctagsSrcDirs, Keys.streams) map genCtags)
 
   val defaultCtagsParams: CtagsParams = CtagsParams(
         executable = "ctags",
         excludes = Seq("log"),
         languages = Seq("scala", "java"),
-        tagFileName = ".tags",
+        tagFileName = "tags",
```

Comment:
I think that https://github.com/ceedubs/sbt-ctags/pull/18/files#r111310762 was getting at this, but this change doesn't seem right to me. Some people use `tags` instead of `.tags` so I think that it should be configurable instead of forcing the `.` in where this is used.